data/json/obsoletion/obsolete_monsters.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "id": "mon_mininuke_hack",
+    "copy-from": "base_drone",
+    "type": "MONSTER",
+    "name": { "str": "mininuke hack" },
+    "description": "Many times as large as a normal manhack, this flying quadcopter drone appears to have a mininuke inside.  If this is targeting you… run.",
+    "diff": 40,
+    "hp": 20,
+    "speed": 150,
+    "color": "light_green_magenta",
+    "revert_to_itype": "bot_mininuke_hack",
+    "starting_ammo": { "mininuke_payload": 1 },
+    "special_attacks": [ [ "KAMIKAZE", 0 ] ],
+    "armor": { "bash": 1, "cut": 6, "bullet": 5 }
+  }
+]
```

Comment:
All of this is unnecessary. We don't do monster obsoletion nowadays. Removed monsters silently transform into breathers.

Reply:
What about the transform target for the inactive mininuke hack?

Reply:
You can test what happens ingame, but I am 99% sure that it'll just spawn a breather without an error. That's what happens to monsters stored inside pet carriers - they still show up with their old name, but when you release them they come out as breathers.

Reply:
It doesn't work.